### DISCUSSION TITLE
Web technologies for publishing

### DISCUSSION THREAD

**Discussion by VictorVenema** [singlesourcepub/community#30] [Ideas]
On the Open Science Fest there was a talk about using web technologies (html, css) for publishing, i.e., for the creation of printed documents, with pages number and all that stuff. I thought that might be interesting for this group, it makes the jump from html to pdf much smaller.

Slides of the talk: https://www.courtbouillon.org/static/presentations/Using%20Web%20Technologies%20for%20Publishing.pdf
The group behind it: https://weasyprint.org

**Comment by xldrkp** [comment]
@VictorVenema Great, didn't know about it. @mrchristian does a lot with publishing from HTML, too, but as far as I know he uses other tools than that. Simon, what do you think about WeasyPrint?
